.github/CODEOWNERS

```diff
@@ -25,6 +27,8 @@ gno.land/*                   @gnolang/tech-staff
 
 # GnoVM/Gnolang.
 gnovm/*                      @gnolang/tech-staff
+gnovm/stdlibs                @jaekwon @moul
+gnovm/pkg/gnolang            @jaekwon @moul
```

Comment:
For GnoVM's core functionalities, @jaekwon and @moul's reviews are mandatory, while for `cmd/` and other packages (e.g., `gnomod`), only @gnolang/tech-staff's approval is required.